codelabs/codelab-03/README.md

```diff
@@ -0,0 +1,143 @@
+# Codelab 3: AWS + S3 + ClounFront
+
+### Due Date
+
+This codelab comes in two parts, both of which are due on *Thursday, February 15th at 11:59:59PM*. This is codelab-03 (part B) which expands upon what was learned in [codelab-02](https://umd-cs-stics.gitbooks.io/cmsc389l-practical-cloud-computing-spring-2018/content/codelabs/codelab-02/) (part A) and introduces CloudFront.
+
+### Goal
+
+In this codelab, you'll get to play around with ClounFront.
+- You'll set up CloudFront via the AWS GUI
+- You'll test latency 
+- You'll issue an invalidation
+
+### Setting Up
+
+Before starting this codelab, run `git pull` in the `389Lspring18` directory to update your local copy of the class repository.
+
+### Tutorial
+
+**It's a good idea to read/skim the entire codelab first so you have an idea of what you're doing. At a minimum, look at the submission section to _avoid having to redo your work_ to obtain screenshots**
+
+#### Back to buckets
+
+Fist, download the image `canyon.jpg`
+![here](../../../media/codelabs/codelab-03/canyon.jpg)
+
+Now, using the [AWS GUI](https://console.aws.amazon.com/console/home?region=us-east-1#), navigate to S3.
+
+From the S3 dashboard, click `Create Bucket`.
+
+In the field labeled *bucket name*, use the format `cmsc389l-#-codelab03`; substituting your directory id for the octothorp(#).
+
+In the field labeled *region*, select `Asia Pacific (Sydney)`.
+
+This is what your form should look like.
+
+![create bucket Page](../../../media/codelabs/codelab-03/create_bucket.png)
+
+Click the `Create` button in the lower-left corner. For the purposes of this tutorial, the default properties and permissions are fine.
+
+Select your new bucket by hovering over its bucket name until highlighted with an underscore; click. This page lets add/view content and configure properties.
```

Comment:
Generally, try to avoid being too explicit with the descriptions

possibly:

Select your newly-created bucket (you have to click the bucket name itself).

Reply:
Same applies to some of the descriptions below